BML_TUX_Project/Packages/bmlTUX/Editor/PackageInstallDetector.cs

```diff
@@ -55,8 +55,7 @@ static Version GetPackageVersion()
         }
 
         static string ProjectFolder => Path.GetDirectoryName(Path.Combine( Application.dataPath, "../" ));
-        static string InstallDataFilePath => Path.Combine(FileLocationSettings.BaseTuxDocumentsFolderPath, ProjectFolder, InstallDataFileName);
-    
+        static string InstallDataFilePath => Path.Combine(ProjectFolder, InstallDataFileName);

```

Comment:
With these changes, I think we need to rename the file now, otherwise it might be confusing sitting at the root of the project. I suggest we rename it to `bmlTux_InstallData.json`